### ISSUE TITLE
Visible labels are not available on editing buttons

### ISSUE THREAD

**Issue by karlgroves** [WordPress/gutenberg#15352]
Visible labels are not available on editing buttons

-   **Severity:** Medium
-   **Affected Populations:**
    -   Low-Vision
    -   Cognitively Impaired
-   **Platform(s):**
    -   All / Universal
-   **Components affected:**
    -   Edit Media

#### Issue description

On the Edit Media page, buttons for editing the selected media image
have accessible names to label them for assistive technology users,
however these names are not available to sighted users.

Since some of the icons are quite cryptic, and there is an ambiguity
between two sets of icons which both appear to mean Rotate. This may be
confusing to users with cognitive disabilities. Such users may find
themselves having to interact with buttons just to figure out what they
do.

#### Remediation Guidance

Add visible text labels to the image editing buttons.

One method, which would be consistent with the rest of the editor, would
be to use keyboard-accessible tooltips for this.

A better method would be to provide a user setting for all icon
controls, so that users can specify whether to show icons-only,
labels-only, or icons and labels together.

**Note:** This issue may be a duplicate with other existing accessibility-related bugs in this project. This issue comes from the Gutenberg accessibility audit, performed by [Tenon](https://www.tenon.io) and funded by [WP Campus](https://wpcampus.org/). This issue is GUT-85 in Tenon's report

**Comment by afercia** [comment]
Worth noting this issue isn't strictly related to Gutenberg. The media views are part of WordPress: this issue should be reported in core Trac (I seem to recall there's a related ticket already) and depends on the introduction of accessible tooltips in core.

**Comment by afercia** [comment]
Moved to Trac: https://core.trac.wordpress.org/ticket/47116
